src/targets/services/helpers.js

```diff
@@ -0,0 +1,57 @@
+import last from 'lodash/last'
+import get from 'lodash/get'
+
+/**
+ * If lastSeq is 0, it's more efficient to fetch all documents.
+ */
+export const fetchChangesOrAll = async (Model, lastSeq) => {
+  if (lastSeq === '0') {
+    const documents = await Model.fetchAll()
+    // fetch last change to have the last_seq
+    const lastChanges = await Model.fetchChanges('', {
```

Comment:
[nit] It would be better to first fetch the `last_seq`, and only after that to get all the documents. It avoids a race condition where a document is created between the two calls, and will never be analyzed for matching.

Reply:
Thanks @nono, I did the change. See https://github.com/cozy/cozy-banks/pull/1552/commits/b2f73f29882311bb73ad95e6bd3c3e151c8ea28c